graph/telemetry/istio/appender/appender.go

```diff
@@ -103,6 +107,24 @@ func ParseAppenders(o graph.TelemetryOptions) []graph.Appender {
 		}
 		appenders = append(appenders, a)
 	}
+	if _, ok := requestedAppenders[AggregateNodeAppenderName]; ok || o.Appenders.All {
+		aggregate := o.NodeOptions.Aggregate
+		aggregateValue := o.NodeOptions.AggregateValue
+		if aggregate == "" {
```

Comment:
One thing perhaps I didn't understand correctly: how will a user change the default aggregate? IIUC "aggregate" is controlled by UI, so, will it be hard-coded in UI, or editable from UI?
Initially I thought that would be something configurable from configmap.

Reply:
@jotak In the initial PR there is no support to change the default aggregate, it is fixed to "request_operation".  The graph request supports a query param that can override the default but the UI is not using that option.  So in this initial PR it is fixed.   If you think we need to make it immediately configurable I can add support.  It seemed OK to require "request_operation" for now only because it's brand new and that's what Istio uses in the example.  let me know...

Reply:
No problem with me, just I wasn't aware.
Also, in the same idea of potential future changes, perhaps we could have multiple custom aggregates and not just a single one, which would add complexity to the model ... but we're not there yet, let's see what people wants :-)